ClientLibrary/lib/src/main/java/com/microsoft/projectoxford/vision/VisionServiceRestClient.java

```diff
@@ -37,8 +37,8 @@
 import com.google.gson.Gson;
 import com.microsoft.projectoxford.vision.contract.AnalysisInDomainResult;
 import com.microsoft.projectoxford.vision.contract.AnalysisResult;
-import com.microsoft.projectoxford.vision.contract.HandwritingOCROperationResult;
-import com.microsoft.projectoxford.vision.contract.HandwritingOCROperation;
+import com.microsoft.projectoxford.vision.contract.HandwritingReconitionOperationResult;
```

Comment:
Change HandwritingReconitionOperationResult 
to          HandwritingRecognitionOperationResult